src/composer/src/main.rs

```diff
@@ -67,10 +68,26 @@ pub fn exec() -> Result<(), String> {
         .rev()
         .collect();
     for c_id in reverse_ids.iter() {
+        
         sys.add_params_iter(&c_id, Parameters::transition_iter(c_id, &sys, &mut build)?);
+        let obj = ElfObject::transition_iter(c_id, &sys, &mut build)?;
         sys.add_objs_iter(&c_id, ElfObject::transition_iter(c_id, &sys, &mut build)?);
         sys.add_invs_iter(&c_id, Invocations::transition_iter(c_id, &sys, &mut build)?);
+        println!("path:{}", obj.get_path());
+        let output = Command::new("python3")
+        .arg("/home/minghwu/work/minghwu/composite/pyelftool_parser/src/analyzer.py")
+        .arg(obj.get_path())
+        .output()
+        .expect("Failed to execute script");
+        if output.status.success() {
+            let stdout = String::from_utf8_lossy(&output.stdout);
+            println!("Script output: {}", stdout);
+        } else {
+            let stderr = String::from_utf8_lossy(&output.stderr);
+            eprintln!("Script error: {}", stderr);
+        }
```

Comment:
For all of this, we'll want to add a pass. This is likely in your other code, so I'm hesitate to say much here.